02-KernelModules_CharacterDevice/Makefile

```
@@ -0,0 +1,7 @@
+obj-m += chrdev.o
+
+all:
+	make -C /home/alex/Downloads/linux-5.0.7 M=$(PWD) modules
+
+clean:
+	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
```

```suggestion
	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
	
```

Makefile - Need fix end of line